lib/std/os.zig

```diff
@@ -3327,3 +3327,106 @@ pub fn getrusage(who: i32) rusage {
         else => unreachable,
     }
 }
+
+pub const termios = extern struct {
+    iflag: tcflag_t,
+    oflag: tcflag_t,
+    cflag: tcflag_t,
+    lflag: tcflag_t,
+    line: cc_t,
+    cc: [NCCS]cc_t,
+    ispeed: speed_t,
+    ospeed: speed_t,
+
+    pub const TermiosError = error {
+        SyscallFailed,
+        UnknownAction,
+    };
+
+    pub fn get(handle: fd_t) TermiosError!@This() {
+        if (builtin.os == .linux) {
+            var term: @This() = undefined;
+            const err = linux.syscall3(linux.SYS_ioctl, @bitCast(usize, @as(isize, handle)), linux.TCGETS, @ptrToInt(&term));
```

Comment:
os.zig shouldn't contain direct syscalls: the syscall wrapper for linux belongs in os/linux.zig. (yes there is currently a violation of this in os.zig already)

Reply:
I see. Yeah I saw said violation and thought it was fine to make direct syscall myself.